lib/game/getUniverseInfo.js

```diff
@@ -0,0 +1,49 @@
+const http = require('../util/http.js').func
+
+exports.required = ['universeIds']
+exports.optional = ['jar']
+
+// Docs
+/**
+ * 🔓 Get the info for a universe.
+ * @category Game
+ * @alias getUniverseInfo
+ * @param {number} universeIds - The ids of the universes.
```

Comment:
Ids implies plural so the parameter should suggest multiple IDs can be used as in the documentation.